internal/connector/internal/presenters/connector_deployment.go

```diff
@@ -25,7 +28,25 @@ func PresentConnectorDeployment(from dbapi.ConnectorDeployment) (private.Connect
 		}
 	}
 
+	// prepare shard metadata
+	var shardMetadataJson map[string]interface{}
```

Comment:
this can be removed

Reply:
yep tnx! It should be something the linter detects?

Reply:
We are in the process of enabling a number of linters. I think it will be detected in the future.